tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -535,7 +535,7 @@ public static function columnSqlProvider()
             ],
             [
                 'id',
-                ['type' => 'uuid'],
```

Comment:
Did you intend to remove CHAR(36) => UUID mapping for mysql?

Reply:
The mapping doesn't work?

Reply:
It is already there at L120

Reply:
Added additional test.